app/views/admin/order_cycles/index.html.haml

```diff
@@ -7,10 +7,10 @@
 = content_for :page_actions do
   - if subscriptions_enabled?
     %li
-      %a.button.icon-plus#new-schedule{ "schedule-dialog" => true }
+      %a.button.icon-plus#new-schedule{ "schedule-dialog" => true, "ofn-with-tip" => t('admin.order_cycles.index.new_schedule_tooltip'), "powertip-location" => 's' }
```

Comment:
Nice!

I think you can shorten this with lazy lookup:
```suggestion
      %a.button.icon-plus#new-schedule{ "schedule-dialog" => true, "ofn-with-tip" => t('.new_schedule_tooltip'), "powertip-location" => 's' }
```

You can shorten the other `t` calls in this file at the same time. :smile:

Reply:
Made changes, the other t call is already on the top level, so should'nt matter